sdks/python/apache_beam/runners/runner.py

```diff
@@ -26,9 +26,22 @@
 import tempfile
 
 
+_path_generator = lambda rs, p: {r.lower(): p + r for r in rs}
```

Comment:
This should be a proper function, with a clear name and clearly named arguments.